Sorry, atm, rather perplexed. It seems that 

1. the test is still failing (although I have re-run it manually clicking the ``re-run`` button), and is claiming that the nginx 1.20 is installed, whereas the ``wanted`` version in the test is 1.26  == fail

```
AssertionError: assert 'nginx version: nginx/1.20.2' == 'nginx version: nginx/1.26.2'
  E
```

2. I have installed the role version from this branch locally and run the playbook against ``ome-train4-ap1``. The result is: although yesterday there was 1.24 nginx, after the run of the playbook I have now 1.20 

```
[root@ome-train4-ap1 pwalczysko]# nginx -v
nginx version: nginx/1.20.1
```

Edit:
Sorry, I must have made a mistake in point 2. above. The role was wrongly cited in the playbook.
Now, after re-run with ``really the role from this branch``, I am having

```
TASK [nginx : system packages | setup upstream nginx stable repo] ************************************
fatal: [134.36.5.145]: FAILED! => {"changed": false, "failures": [], "msg": "Unknown Error occurred: Transaction test error:\n  file /etc/logrotate.d/nginx from install of nginx-2:1.26.2-2.el9.ngx.x86_64 conflicts with file from package nginx-core-2:1.20.1-20.el9.x86_64\n  file /etc/nginx/mime.types from install of nginx-2:1.26.2-2.el9.ngx.x86_64 conflicts with file from package nginx-core-2:1.20.1-20.el9.x86_64\n  file /etc/nginx/nginx.conf from install of nginx-2:1.26.2-2.el9.ngx.x86_64 conflicts with file from package nginx-core-2:1.20.1-20.el9.x86_64\n  file /usr/sbin/nginx from install of nginx-2:1.26.2-2.el9.ngx.x86_64 conflicts with file from package nginx-core-2:1.20.1-20.el9.x86_64\n  file /var/log/nginx from install of nginx-2:1.26.2-2.el9.ngx.x86_64 conflicts with file from package nginx-core-2:1.20.1-20.el9.x86_64\n", "rc": 1, "results": []}
```

--- a/.github/workflows/molecule.yml
+++ b/.github/workflows/molecule.yml
@@ -53,4 +53,3 @@ jobs:
         with:
           galaxy_api_key: ${{ secrets.GALAXY_API_KEY }}
           galaxy_version: ${{  github.ref_name }}
-

--- a/molecule/rockylinux9/molecule.yml
+++ b/molecule/rockylinux9/molecule.yml
@@ -48,7 +48,7 @@ provisioner:
         nginx_keep_default_configs: true
         nginx_stable_repo: false
         # Current mainline is 1.15.9
-        nginx_version: 1.20.2
+        nginx_version: 1.26.2
         nginx_logrotate_interval: weekly
         nginx_logrotate_backlog_size: 5
       nginx-disabled:

--- a/molecule/rockylinux9/molecule.yml
+++ b/molecule/rockylinux9/molecule.yml
@@ -34,7 +34,7 @@ platforms:
     tmpfs:
       - /sys/fs/cgroup
     groups:
-        - extra_options
+      - extra_options
 provisioner:
   name: ansible
   playbooks:

--- a/tasks/debian.yaml
+++ b/tasks/debian.yaml
@@ -1,17 +1,4 @@
 ---
-# - name: system packages | add nginx apt key
-#   become: true
-#   ansible.builtin.apt_key:
-#     keyring: /usr/share/keyrings/nginx-archive-keyring.gpg
-#     url:  https://nginx.org/keys/nginx_signing.key
-#     id: ABF5BD827BD9BF62
-
-
-# - name: system packages | Add nginx stable repository into sources list
-#   ansible.builtin.apt_repository:
-#     repo: "deb [arch=amd64,arm64 signed-by=/usr/share/keyrings/nginx-archive-keyring.gpg] http://nginx.org/packages/ubuntu jammy nginx"
-#     state: present
-
 - name: system packages | install nginx
   become: true
   ansible.builtin.apt:
@@ -21,6 +8,6 @@
 
 - name: nginx | remove default config
   become: true
-  file:
+  ansible.builtin.file:
     path: /etc/nginx/sites-enabled/default
     state: absent

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -16,7 +16,7 @@
 
 - name: nginx | configure logrotate
   become: true
-  template:
+  ansible.builtin.template:
     backup: false
     dest: /etc/logrotate.d/nginx
     src: logrotated-nginx.j2

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -1,15 +1,15 @@
 ---
 # tasks file for roles/nginx
 
-- import_tasks: redhat.yaml
+- ansible.builtin.import_tasks: redhat.yaml
   when: ansible_os_family | lower == 'redhat'
 
-- import_tasks: debian.yaml
+- ansible.builtin.import_tasks: debian.yaml
   when: ansible_os_family | lower == 'debian'
 
 - name: nginx | running
   become: true
-  service:
+  ansible.builtin.service:
     name: nginx
     state: started
   when: nginx_systemd_setup

--- a/tasks/redhat.yaml
+++ b/tasks/redhat.yaml
@@ -35,7 +35,7 @@
 # just in case
 - name: nginx | remove default config
   become: true
-  copy:
+  ansible.builtin.copy:
     content: "# This file is intentionally blank (Ansible)"
     dest: /etc/nginx/conf.d/{{ item }}
     mode: 0644

--- a/tasks/redhat.yaml
+++ b/tasks/redhat.yaml
@@ -3,13 +3,14 @@
   become: true
   ansible.builtin.rpm_key:
     state: present
-    key:  https://nginx.org/keys/nginx_signing.key
+    key: https://nginx.org/keys/nginx_signing.key
 
 - name: system packages | setup upstream nginx stable repo
   become: true
   ansible.builtin.dnf:
     update_cache: true
-    name: "https://nginx.org/packages/centos/9/x86_64/RPMS/nginx-1.26.2-2.el9.ngx.x86_64.rpm"
+    name: "https://nginx.org/packages/centos/9/x86_64/RPMS/\
+           nginx-1.26.2-2.el9.ngx.x86_64.rpm"
     state: present
   when: not nginx_version
 
@@ -17,7 +18,8 @@
   become: true
   ansible.builtin.dnf:
     update_cache: true
-    name: "https://nginx.org/packages/centos/9/x86_64/RPMS/nginx-{{ nginx_version }}-1.el9.ngx.x86_64.rpm"
+    name: "https://nginx.org/packages/centos/9/x86_64/RPMS/\
+           nginx-{{ nginx_version }}-1.el9.ngx.x86_64.rpm"
     state: present
   when: nginx_version
 

